x/mongo/driver/batch_cursor.go

```diff
@@ -266,7 +266,7 @@ func (bc *BatchCursor) getMore(ctx context.Context) {
 		},
 		Database:   bc.database,
 		Deployment: SingleServerDeployment{Server: bc.server},
-		ProcessResponseFn: func(response bsoncore.Document, srvr Server, desc description.Server) error {
+		ProcessResponseFn: func(response bsoncore.Document, srvr Server, desc description.Server, currIndex int) error {
```

Comment:
Here and in every other operation file besides `update.go`: this field is ignored so we should express that by marking it as `_` in the function signature rather than giving it a name

Reply:
done